.github/workflows/bookkeeper.yml

```diff
@@ -0,0 +1,32 @@
+name: Pulsarctl BookKeeper Unit Tests Check
+on: [pull_request]
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.12
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.12
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v1
+
+    - name: Run bookKeeper service
+      run: |
+        pushd test/docker
+        docker-compose pull
+        docker-compose up -d
+        popd
+        sleep 10
```

Comment:
using `sleep` is a very bad practice in testing. It is non-deterministic and causes flakiness.
what you should do is to check if the bookkeeper cluster is ready.